tests/client/components/Users/UserActions.tests.js

```diff
@@ -142,7 +142,7 @@ describe('#Client-Components-UserActions', () => {
     checkMenuItems(menuItems, targets);
   });
 
-  it('should not render reset password, if disabled in userFields', () => {
+  it.only('should not render reset password, if disabled in userFields', () => {
```

Comment:
Could you please remove this `.only`?

Reply:
Good spot, sorry about that. It should be fixed now, waiting for the build to complete.